exercises/matching-brackets/runtests.jl

```diff
@@ -0,0 +1,23 @@
+using Test
+
+include("matching-brackets.jl")
+
+@testset "Determine if brackets in expression are correctly matched" begin
+    @test matching_brackets("[]")::Bool == true  
```

Comment:
- `::Bool` at the end is not necessary
- `matching_brackets("[]")` already returns a `Bool`. The `== true` or `== false` are not necessary.

Two examples:

```julia
    @test  matching_brackets("[]")
    @test !matching_brackets("[[")
```

I like aligning the function name in cases like this to make it easier to see which tests are expected to evaluate to false, and which are expected to be true.